lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -513,7 +569,7 @@ if (require.main === module) {
 }
 
 module.exports = {
-  computeDescription,
+  parseUIStrings,
   createPsuedoLocaleStrings,
-  convertMessageToPlaceholders: convertMessageToCtc,
+  convertMessageToCtc,
```

Comment:
it didn't make sense to rename this just for exporting for tests, but missed in the review for #9114 :)